samples/timer_overlay/index.js

```diff
@@ -22,6 +23,18 @@ document.addEventListener("DOMContentLoaded", () => {
   let timerRunning = false;
   let currentTime;
 
+  let height = 720;
+  let width = 1280;
+
+  if (window.innerHeight < 500) {
+    height = 250;
+    width = 640;
+  }
+  else if (window.innerHeight < 1000) {
+    height = 400;
+    width = 350;
+  }
+
```

Comment:
What's the logic here? Why are the aspect ratios varying so much? Is there some sort of portrait vs landscape hackiness going on here?

Reply:
Updated

Reply:
Yes this is to try and capture mobile vs desktop and portrait vs landscape